nextstrain.yml

```diff
@@ -1,25 +1,16 @@
 name: nextstrain
 channels:
+- conda-forge
 - bioconda
 - defaults
 dependencies:
-- cvxopt
-- datrie
-- fasttree
-- iqtree
-- mafft
-- opencv
-- pandas
-- psutil
-- python=3.6*
-- nodejs=10
-- raxml
-- vcftools
+- augur
+- nextalign=0.1.6
+- nextstrain-cli
+- python>=3.6*
+- nodejs>=10
 - git
 - pip
 - pip:
   - awscli
```

Comment:
Have you tried https://anaconda.org/conda-forge/awscli?

Reply:
I just tried this today, but when I created a new environment with mamba, an old version of Snakemake (5.5.*) got installed instead of the latest version (although [we currently require Snakemake >=5.4.0](https://bioconda.github.io/recipes/augur/README.html) for Augur, we really prefer the latest).

When I tried to install with conda instead, the environment took much much longer to solve. When I added Snakemake to this conda environment to override Augur's requirements and set the min version to 5.26.*, the conda solver took a long time to solve the environment, but mamba quickly solved it and installed Snakemake 6.0.2 and awscli 1.19.20.

I think we eventually want to source all of our dependencies from conda channels, but for this PR I'd like to stick with awscli as a pip package.